Sources/SwiftSyntaxBuilderGeneration/Templates/TokensFile.swift

```diff
@@ -51,7 +51,30 @@ let tokensFile = SourceFile {
             FunctionCallExpr(MemberAccessExpr(base: "TokenSyntax", name: "\(token.swiftKind)Token"))
           }
 
-          createTokenSyntaxPatternBinding("`\(token.name.withFirstCharacterLowercased)`", accessor: accessor)
+          createTokenSyntaxPatternBinding(token.name.withFirstCharacterLowercased.backticked, accessor: accessor)
+        }
+      } else {
+        let signature = FunctionSignature(
+          input: ParameterClause {
+            FunctionParameter(
+              attributes: nil,
+              firstName: .wildcard,
+              secondName: .identifier("text"),
+              colon: .colon,
+              type: "String"
+            )
+          },
+          output: "TokenSyntax"
+        )
+
+        FunctionDecl(
+          modifiers: [TokenSyntax.static],
+          identifier: .identifier(token.name.withFirstCharacterLowercased.backticked),
+          signature: signature
+        ) {
+          FunctionCallExpr(MemberAccessExpr(base: "SyntaxFactory", name: "make\(token.name)")) {
```

Comment:
Could you migrate this to not use `SyntaxFactory`, which I recently deprecated. I think this should be something like
```suggestion
          FunctionCallExpr(MemberAccessExpr(base: "TokenSyntax", name: token.name)) {
```

Reply:
Ah whoops, looks like I accidentally picked the wrong side when resolving the conflict. The generated `Tokens.swift` shouldn't have changed at all.